index.html

```diff
@@ -6024,6 +6032,9 @@ <h4>JsonLdErrorCode</h4>
         <dt><dfn>invalid vocab mapping</dfn></dt>
         <dd>An invalid <a>vocabulary mapping</a> has been detected,
           i.e., it is neither an <a>IRI</a> nor <code>null</code>.</dd>
+        <dt class="changed">IRI confused with prefix</dt>
+        <dd>When compacting an <a>absolute IRI</a> would result in an <a>absolute IRI</a>
+          which could be confused with a compact IRI (because its <a data-cite="RFC3986#section-3.1">IRI scheme</a> matches a <a>term definition</a> and it has no <a data-cite="RFC3986#section-3.1">IRI authority</a>).</dd>
```

Comment:
This is normative text, so should we *again* specify that the prefix flag must be set to `true` (which would be very verbose)?

Reply:
I'm not sure it's necessary here, as we describe elsewhere in the algorithm that the _prefix flag_ must be `true`. This is just saying that an error is generated because the IRI scheme matches a term; the fact that the _prefix flag_ is `true` is implicit.

Reply:
Fine with me.